src/legacy/ui/public/vis/components/tooltip/_pointseries_tooltip_formatter.js

```diff
@@ -61,8 +60,7 @@ export function PointSeriesTooltipFormatterProvider($compile, $rootScope) {
       }
       if (datum.series && datum.parent) {
         const dimension = datum.parent;
-        const seriesFormatter = getFormat(dimension.format);
-        addDetail(dimension.title, seriesFormatter.convert(datum.series));
+        addDetail(dimension.title, datum.series);
```

Comment:
what preformats this values ? 
are we sure they will always be preformated ?
(it behaves diff than all the other dimensions :| )

Reply:
that was reported on the PR description:
https://github.com/elastic/kibana/blob/792286962193f585f332cad7bc2fb280256dd1d3/src/legacy/ui/public/agg_response/point_series/_get_point.js#L73-L83